frontend/ui/uimanager.lua

```diff
@@ -391,8 +391,14 @@ function UIManager:show(widget, refreshtype, refreshregion, x, y, refreshdither)
     -- put this window on top of the toppest non-modal window
     for i = #self._window_stack, 0, -1 do
         local top_window = self._window_stack[i]
-        -- skip modal window
-        if widget.modal or not top_window or not top_window.widget.modal then
+        -- toasts are stacked at top over other toasts,
+        -- then are modals, and then other widgets
```

Comment:
`then come modals`

Reply:
My consolation is that my coding is better than my english :) 
Please double check my logic on toasts & modals :)

Reply:
Looks fine AFAICT ;).